src/components/SelfUser.vue

```diff
@@ -31,13 +34,16 @@ export default {
                 str = '+' + str;
             }
 
-            return str;
+            return 'User modes: ' + str;
         },
         netUser() {
             return this.network.ircClient.user;
         },
     },
     methods: {
+        hide: function hide() {
+            this.parent.selfuser_open = false;
```

Comment:
Child components should never interact with its parent directly and passing it in as a prop is just a long about way of doing that. It's better to emit a `close` event and use `@close="selfuser_open=false"` on the parent instead